core/trino-main/src/test/java/io/trino/server/security/oauth2/TestingHydraIdentityProvider.java

```diff
@@ -63,6 +63,7 @@
         implements AutoCloseable
```

Comment:
Can you explain in the commit message why?

Reply:
I've updated the commit message.